corehq/apps/repeaters/tests/test_dbaccessors.py

```diff
@@ -94,7 +94,7 @@ def test_get_failure_repeat_record_count(self):
 
     def test_get_repeat_record_count_with_state_and_no_repeater(self):
         count = get_repeat_record_count(self.domain, state=RECORD_PENDING_STATE)
-        self.assertEqual(count, 6)
+        self.assertEqual(count, 3)
```

Comment:
Ok this should actually be 3, as that matches `get_paged_repeat_records()` with the same arguments:
https://github.com/dimagi/commcare-hq/pull/15991/commits/9a90ee2d715ffaec7314dca7492cb06c0d966ea3#diff-a556cb8edac1a5f46167e86c50f0eeafR117